website/content/en/docs/User Guide/Learn Cloud Operations/cloud_error_reporting.md

```diff
@@ -37,4 +38,50 @@ Click **View Logs** for one of the samples to see the log messages that match th
 
 You can expand any of the messages that matches the filter to see the full stack trace:
 
-![image](/docs/images/user-guide/35-logdet.png)
\ No newline at end of file
+![image](/docs/images/user-guide/35-logdet.png)
+
+#### Errors Manufacturing 
+
+There are several ways in which you can experiment with Error Reporting tool and manufacture errors that will be reported and displayed in the UI. For the purpose of this demonstration we will use 2 tools that are coming with Cloud Operations Sandbox: Load Generator and SRE Recipes to simulate a situation that `Sandbox break`.
+
+To simulate requests using the load generator we can use the UI or `sandboxctl`
+
+```
+$sandboxctl loadgen step
+Redeploying Loadgenerator...
+Loadgenerator deployed using step pattern
+Loadgenerator web UI: http://<ExampleIP>
+```
+
+Then to `break` the service we will use sre-recipes (recipe2)
```

Comment:
It might be a good idea to explain what sre-recipes is here (or above)

Reply:
Between branches the SRE Recipes update was missed I added it again to the User Guide and  linked here. Please have another look.